### ISSUE TITLE
how can I split batches for training  [Feature Request]

### ISSUE THREAD

**Issue by shubjain26** [ildoonet/tf-pose-estimation#155]
Is there any way I can implement caffe like iter_size feature to split a batch into multiple for training.
The problem is I cant have higher batch size due to GPU memory. 

this is my training loss curve while training mobilenet on coco data with batch size of 32. Assuming increasing the batch size should help need help in implementation.
 
![image](https://user-images.githubusercontent.com/16733195/39488780-ee189f2c-4da0-11e8-98c4-339b4f8d6973.png)

Thanks,

**Comment by ildoonet** [comment]
This already have mini-batch implementations. What is your specific demand?

**Comment by shubjain26** [comment]
ohk, how can I use mini-batch to train with higher batch size ? 
currently highest I can have is 32 in my machine.  I want to train with 96/128 batch size.

I tried using this [link](https://stackoverflow.com/a/42222976/9163608) , but its very slow.
